package.json

```
@@ -1,45 +1,42 @@
 {
   "private": true,
-  "workspaces": [
-    "packages/*",
-    "draft-packages/*"
-  ],
   "engines": {
     "node": ">=v18.16.0"
   },
   "scripts": {
-    "build": "yarn turbo build",
-    "clean": "yarn turbo clean && yarn storybook:clean && rimraf node_modules",
-    "test": "jest && yarn test:pkg:aio",
-    "test:pkg:aio": "yarn pkg:aio test && yarn pkg:aio build",
+    "preinstall": "npx only-allow pnpm",
+    "build": "pnpm turbo build",
+    "clean": "pnpm turbo clean && pnpm storybook:clean && rimraf node_modules",
+    "test": "pnpm turbo test",
+    "test:pkg:aio": "pnpm pkg:aio build",
     "test:ci": "jest --ci",
-    "test:treeshake": "yarn turbo test:treeshake",
-    "compile": "tsc && yarn pkg:aio compile",
+    "test:treeshake": "pnpm turbo test:treeshake",
+    "compile": "tsc && pnpm pkg:aio compile",
     "storybook": "storybook dev -p 6006 -c storybook",
-    "storybook:preview": "NODE_ENV=production yarn storybook --docs",
+    "storybook:preview": "NODE_ENV=production pnpm storybook --docs",
     "storybook:clean": "rm -rf storybook/public",
     "storybook:build": "storybook build -c storybook -o storybook/public --docs",
     "test:storybook": "test-storybook --config-dir storybook",
     "jest:debug": "node --inspect-brk node_modules/.bin/jest --runInBand --no-cache",
-    "lint": "yarn lint:ts && yarn lint:md && yarn lint:format && yarn lint:styles",
-    "eslint-config": "yarn eslint -c .eslintrc.js --max-warnings=0",
-    "lint:ts": "yarn eslint-config '**/*.{ts,tsx}'",
-    "lint:md": "yarn eslint-config '**/*.{md,mdx}'",
-    "lint:format": "yarn prettier --check '**/*'",
-    "lint:styles": "yarn stylelint '**/*.scss'",
-    "lint:fix": "yarn lint:styles --fix && yarn lint:ts --fix && yarn lint:md --fix && yarn lint:format --write",
-    "reset": "yarn clean && yarn install --force",
+    "lint": "pnpm lint:ts && pnpm lint:md && pnpm lint:format && pnpm lint:styles",
+    "eslint-config": "pnpm eslint -c .eslintrc.js --max-warnings=0",
+    "lint:ts": "pnpm eslint-config '**/*.{ts,tsx}'",
+    "lint:md": "pnpm eslint-config '**/*.{md,mdx}'",
+    "lint:format": "pnpm prettier --check '**/*'",
+    "lint:styles": "pnpm stylelint '**/*.scss'",
+    "lint:fix": "pnpm lint:styles --fix && pnpm lint:ts --fix && pnpm lint:md --fix && pnpm lint:format --write",
+    "reset": "pnpm clean && pnpm install --force",
     "chromatic": "chromatic",
-    "commit": "yarn changeset",
+    "commit": "pnpm changeset",
     "plop": "plop",
-    "pkg:aio": "yarn workspace @kaizen/components",
-    "ci:version": "yarn turbo prepublishOnly && yarn changeset version",
-    "ci:publish": "yarn turbo prepublishOnly && yarn changeset publish",
-    "update-icons": "yarn pkg:aio update-icons"
+    "pkg:aio": "pnpm --filter @kaizen/components",
```

```suggestion
    "pkg:aio": "pnpm -F @kaizen/components",
```
learnt this nifty trick